OCA-UFCG/oca-frontend
# Add a News Carousel

This PR introduces the creation of the NewsCarousel component and its integration into the new CarouselSection. The NewsCarousel dynamically displays news items with an auto-advancing feature and interactive navigation buttons. 

How to Test:
Add News in CMS
Check if the section appears on the main page with only one News
Add more News
Veirfy the news transition
